File: biochemical.analysis.frontend/src/components/blood/BloodAnalysis.js
import React, { Component } from 'react'
import Blood from './analysis/Blood'
import BiochemicalParameters from './analysis/BiochemicalParameters';
import Fat from './analysis/Fat';
import OtherParameters from './analysis/OtherParameters';

class BloodAnalysis extends Component {
    render() {
        return (
            <div className="row">
                <div className="container mt-4 mx-2 pb-2">
                    <h6 className="mt-2">
                        Blood analysis
                    </h6>
                    <div className="row">
                        <Blood/>
                        <BiochemicalParameters/>
                        <Fat/>
                        <OtherParameters/>
                    </div>
                </div>
            </div>
        )
    }
}

export default BloodAnalysis;

File: biochemical.analysis.backend/biochemical.analysis/src/main/java/sbz/biochemical/analysis/model/blood/BloodAnalysis.java
package sbz.biochemical.analysis.model.blood;

import sbz.biochemical.analysis.model.blood.analysis.BiochemicalParameters;
import sbz.biochemical.analysis.model.blood.analysis.Blood;
import sbz.biochemical.analysis.model.blood.analysis.Fat;
import sbz.biochemical.analysis.model.blood.analysis.OtherParameters;

import java.io.Serializable;

public class BloodAnalysis implements Serializable {

    private Blood blood;
    private BiochemicalParameters biochemicalParameters;
    private Fat fat;
    private OtherParameters otherParameters;

    public BloodAnalysis() {
    }

    public BloodAnalysis(Blood blood, BiochemicalParameters biochemicalParameters, Fat fat, OtherParameters otherParameters) {
        this.blood = blood;
        this.biochemicalParameters = biochemicalParameters;
        this.fat = fat;
        this.otherParameters = otherParameters;
    }

    public Blood getBlood() {
        return blood;
    }

    public void setBlood(Blood blood) {
        this.blood = blood;
    }

    public BiochemicalParameters getBiochemicalParameters() {
        return biochemicalParameters;
    }

    public void setBiochemicalParameters(BiochemicalParameters biochemicalParameters) {
        this.biochemicalParameters = biochemicalParameters;
    }

    public Fat getFat() {
        return fat;
    }

    public void setFat(Fat fat) {
        this.fat = fat;
    }

    public OtherParameters getOtherParameters() {
        return otherParameters;
    }

    public void setOtherParameters(OtherParameters otherParameters) {
        this.otherParameters = otherParameters;
    }
}
